lib/platform.py

```diff
@@ -143,6 +143,11 @@ def info():
   else:
     os='linux'
 
+  try:
+    memory = int(os.sysconf('SC_PAGE_SIZE') * os.sysconf('SC_PHYS_PAGES')/(1024.**3) + 0.5)
+  except:
+    memory = 2
+
```

Comment:
This always returns `2` because `os.sysconf` will through an exception that `os` is a `str` object and does not have `sysconf` attribute...

The problem is that the `os` module is overshadowed by the `os` assignment at line 142/144.

Reply:
We don't need the `try...except` block since you are not using "subprocess" anymore.